drivers/include/mtd.h

```diff
@@ -105,13 +126,22 @@ typedef struct {
  * @brief   MTD driver can write any data to the storage without erasing it first.
  *
  * If this is set, a write completely overrides the previous values.
- *
- * Its absence makes no statement on whether or not writes to memory areas that
- * have been written to previously are allowed, and if so, whether previously
- * written bits should be written again or not written.
  */
 #define MTD_DRIVER_FLAG_DIRECT_WRITE    (1 << 0)
 
+/**
+ * @brief   MTD driver supports arbitrary clearing overwrites
+ *
+ * If this is set, (arbitrarily) many writes are permitted per write size, and
+ * the result is the old value bitwise-AND the written value.
+ *
+ * This property is common for NAND flash memories. (By comparison, the NOR
+ * flash often used internally by MCUs may not allow overwrites, or may allow
```

Comment:
```suggestion
 * This property is common for managed flash memories. (By comparison, the raw
 * flash often used internally by MCUs may not allow overwrites, or may allow
```

NAND flash is actually much more fickle than NOR flash, much so that error correction is *required* and writes to neighboring cells can affect *unrelated cells*.
This makes a [flash-translation layer](https://github.com/dlbeer/dhara) pretty much mandatory when dealing with NAND flash.